scripts/bash/run_enceladus.sh

```diff
@@ -83,204 +84,207 @@ do
     ;;
-    --empty-values-as-nulls)
+  --empty-values-as-nulls)
     EMPTY_VALUES_AS_NULLS="$2"
     shift 2 # past argument and value
     ;;
-    --null-value)
+  --null-value)
     NULL_VALUE="$2"
     shift 2 # past argument and value
     ;;
-    --cobol-encoding)
+  --cobol-encoding)
     COBOL_ENCODING="$2"
     shift 2 # past argument and value
     ;;
-    --cobol-is-text)
+  --cobol-is-text)
     COBOL_IS_TEXT="$2"
     shift 2 # past argument and value
     ;;
-    --cobol-trimming-policy)
+  --cobol-trimming-policy)
     COBOL_TRIMMING_POLICY="$2"
     shift 2 # past argument and value
     ;;
-    --is-xcom)
+  --is-xcom)
     IS_XCOM="$2"
     shift 2 # past argument and value
     ;;
-    --copybook)
+  --copybook)
     COPYBOOK="$2"
     shift 2 # past argument and value
     ;;
-    --mapping-table-pattern)
+  --mapping-table-pattern)
     MAPPING_TABLE_PATTERN="$2"
     shift 2 # past argument and value
     ;;
-    --std-hdfs-path)
+  --std-hdfs-path)
     STD_HDFS_PATH="$2"
     shift 2 # past argument and value
     ;;
-    --debug-set-raw-path)
+  --debug-set-raw-path)
     DEBUG_SET_RAW_PATH="$2"
     shift 2 # past argument and value
     ;;
-    --menas-credentials-file)
+  --menas-credentials-file)
     MENAS_CREDENTIALS_FILE="$2"
     shift 2 # past argument and value
     ;;
-    --menas-auth-keytab)
+  --menas-auth-keytab)
     MENAS_AUTH_KEYTAB="$2"
     shift 2 # past argument and value
     ;;
-    --experimental-mapping-rule)
+  --experimental-mapping-rule)
     EXPERIMENTAL_MAPPING_RULE="$2"
     shift 2 # past argument and value
     ;;
-    --catalyst-workaround)
+  --catalyst-workaround)
     CATALYST_WORKAROUND="$2"
     shift 2 # past argument and value
     ;;
-    --autoclean-std-folder)
+  --autoclean-std-folder)
     AUTOCLEAN_STD_FOLDER="$2"
     shift 2 # past argument and value
     ;;
-    --persist-storage-level)
+  --persist-storage-level)
     PERSIST_STORAGE_LEVEL="$2"
     shift 2 # past argument and value
     ;;
-    --conf-spark-dynamicAllocation-minExecutors)
+  --conf-spark-dynamicAllocation-minExecutors)
     DRA_MIN_EXECUTORS="$2"
     shift 2 # past argument and value
     ;;
-    --conf-spark-dynamicAllocation-maxExecutors)
+  --conf-spark-dynamicAllocation-maxExecutors)
     DRA_MAX_EXECUTORS="$2"
     shift 2 # past argument and value
     ;;
-    --conf-spark-dynamicAllocation-executorAllocationRatio)
+  --conf-spark-dynamicAllocation-executorAllocationRatio)
     DRA_ALLOCATION_RATIO="$2"
     shift 2 # past argument and value
     ;;
-    --conf-spark-sql-adaptive-shuffle-targetPostShuffleInputSize)
+  --conf-spark-sql-adaptive-shuffle-targetPostShuffleInputSize)
     ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE="$2"
     shift 2 # past argument and value
     ;;
-    --set-dra)
+  --set-dra)
     DRA_ENABLED="$2"
     shift 2 # past argument and value
     ;;
-    --help)
+  --help)
     HELP_CALL="1"
     shift # past argument
     ;;
-    *)    # unknown option
+  --asynchronous)
```

Comment:
100%. In GH Pages?

Reply:
Sure GH Pages.

But, and please take it as a suggestion only, as we detect the `--help` parameter there, it shouldn't be that hard to print the Helper script specific parameters and their explanation right before calling `spark-submit SparkJobs.jar --help`. And it would look OK. What do you think?

Reply:
That sounds like a better idea than mine. Will look into adding it to --help printout

Reply:
I went on a bit of an experimental journey here. Adding a class to spark jobs and running only help.

Reply:
Why separate class?

Reply:
Do you have a better idea on how to invoke just the help of scopt?

Reply:
I would just call a default class help (like Standardization)

Reply:
Honestly, I don't like the fact that we do (and could extend in future) validations before scopt gets the stuff. I liked that it is directly going to scopt parser.
I reverted back to using only the default class so you can compare.

@benedeki we had this talk as well if I am not mistaken so let me rope you in.

Reply:
Yup. I wondered myself, about the benefit. But was not hard to convince. This seems elegant enough, so we don't have to worry about the overhead and requirements of the full spark job preparations.
On the other hand... 🤔  this works only when using the _Helper scripts_. When using direct `spark-submit` the problem is still there.
But addressing this is getting fat out of scope of the original fix.

Reply:
OK, I see you removed the class. Solution as well. 👍

Reply:
Yes, I think that the general, even non helper_scripts --help, should be able to execute without any config validations or others in the way, so I am going to create a separate issue for that.